src/routing/Router.svelte

```diff
@@ -0,0 +1,95 @@
+<script context="module">
+  // We could grab the value of currentQuery here rather than requiring that it
```

Comment:
If you wanted to get an up-to-date store value, you could use `get` I guess. But yeah this won't update the hrefs elsewhere when the querystring changes. That's an important part I think, since this appends the current query to the rest of the path

Reply:
> If you wanted to get an up-to-date store value, you could use `get` I guess. But yeah this won't update the hrefs elsewhere when the querystring changes. That's an important part I think, since this appends the current query to the rest of the path

Exactly. That's the part I couldn't figure out. Argument it is!